tutorials/platform/android/native_debug_symbols.rst

```diff
@@ -0,0 +1,75 @@
+.. _doc_native_debug_symbols:
+
+Native Debug Symbols
+====================
+
+When your game crashes on Android, you often see obfuscated stack traces in Play Console or other crash reporting tools like Firebase Crashlytics.
+To make these stack traces human-readable (symbolicated), you need native debug symbols that correspond to your game’s exported build.
+
+Godot now provides downloadable native debug symbols for each official export template. However, For custom template
```

Comment:
```suggestion
Godot now provides downloadable native debug symbols for each official export template.
```

I'd stop there and have it followed by the section that describes how to access the native debug symbols for the official templates.
The process for the custom template can then get its own section afterward.